cli/src/main/package/bin/ide.bat

```diff
@@ -5,6 +5,13 @@ Set _fBGreen=[92m
 Set _fBRed=[91m
 Set _RESET=[0m
 
+rem Add Git PATH entries for CMD - https://github.com/devonfw/IDEasy/issues/764
+for %%H in ( HKEY_LOCAL_MACHINE HKEY_CURRENT_USER ) do for /F "usebackq tokens=2*" %%O in (`call "%SystemRoot%"\system32\reg.exe query "%%H\Software\GitForWindows" /v "InstallPath" 2^>nul ^| "%SystemRoot%\system32\findstr.exe" REG_SZ`) do set GIT_HOME=%%P
+
+if exist "%GIT_HOME%\bin\bash.exe" (
```

Comment:
Do we have to check for Git-Bash?

Reply:
We need to do such check in case the registry entry was not found and `%GIT_HOME%` is empty or containing nonsense.
We could check for a different thing than bash here (e.g. `\usr\bin\basename.exe`), but it will not really make a difference and may even be more fragile for future git releases.

Additionally, we should check if the entries we plan to add to the PATH are already on the PATH avoiding to create duplicates on every call.
Unlike bash or zsh the CMD does not have a concept of sub-environments so we are continuously extending the `PATH` with every call making it longer and longer until we finally exceed some limit and cause an error.
Please also note that when installing git for windows you get an option in the installer GUI asking if you want to install globally into the system or not:
![image](https://github.com/user-attachments/assets/42b6ac42-d0de-4516-b03d-0e1993bf76d4)
The default is the first option that is actually causing this problem we are trying to workaround.
So ideally we write CMD code here that only adds the missing PATH entries if they are not already present and that does not break if something is going wrong with this workaround to avoid blocking everything else.

Reply:
https://superuser.com/questions/480238/cmd-append-to-path-without-duplicating-it

Reply:
Done, I had to create two IF blocks where each entry is added to the PATH, because the PATH doesn't update in the same block. With that approach, we dont, have to use DelayedExpansion.